conda/recipes/mantid/meta.yaml

```diff
@@ -65,6 +65,7 @@ requirements:
     - scipy
     - openssl
     - euphonic=0.6.*
+    - pint<0.22  # version 0.22 introduces a breaking change to euphonic 0.6.*
```

Comment:
Shall we move the `euphonic` and `pint` pins to the `conda_build_config.yaml` folder for consistency?

Reply:
yes that would be good

Reply:
Is this file now the place to make dependency changes, or do we still need to also make a PR against the conda-forge feedstock?

Reply:
By the conda-forge feedstock, do you mean the old `conda-recipes` repo? I believe this repo has now been moved into the mantid repo @thomashampson ?

Reply:
Yes that's the one, this is where the dependency was added initially https://github.com/mantidproject/conda-recipes/pull/74